test/tests/functional/pbs_multi_sched.py

```diff
@@ -580,6 +580,60 @@ def test_run_limts_per_scheduler(self):
         jc = "Not Running: User has reached server running job limit."
         self.server.expect(JOB, {'comment': jc}, id=jid4)
 
+    def test_list_multi_sched(self):
+        """
+        Test to verify that qmgr list sched works when multiple schedulers are present
+        """
+
+        self.setup_sc1()
+        self.setup_sc2()
+        self.setup_sc3()
+
+        self.server.manager(MGR_CMD_LIST, SCHED, expect=True)
+
+        self.server.manager(MGR_CMD_LIST, SCHED, id="default", expect=True)
+
+        self.server.manager(MGR_CMD_LIST, SCHED, id="sc1", expect=True)
+
+        self.server.manager(MGR_CMD_LIST, SCHED, id="sc2", expect=True)
+
+        self.server.manager(MGR_CMD_LIST, SCHED, id="sc3", expect=True)
+
+        try:
+            self.server.manager(MGR_CMD_LIST, SCHED, id="invalid_scname")
+        except PbsManagerError as e:
+            err_msg = "Unknown Scheduler"
+            self.assertTrue(err_msg in e.msg[0],
+                            "Error message is not expected")
+
+        # delete sc3 sched
+        self.server.manager(MGR_CMD_DELETE, SCHED, id="sc3", sudo=True)
+
+        try:
+            self.server.manager(MGR_CMD_LIST, SCHED, id="sc3")
+        except PbsManagerError as e:
+            err_msg = "Unknown Scheduler"
+            self.assertTrue(err_msg in e.msg[0],
+                            "Error message is not expected")
+
+        self.server.manager(MGR_CMD_LIST, SCHED, expect=True)
+
+        self.server.manager(MGR_CMD_LIST, SCHED, id="default", expect=True)
+
+        self.server.manager(MGR_CMD_LIST, SCHED, id="sc1", expect=True)
```

Comment:
What's the rationale behind these manager list calls? You already called them at the beginning of the function.

Reply:
Just verifying they all work after deleting sc3 here 
https://github.com/Shrini-h/pbspro/blob/e27d7ffa81f8fc6c44d5a54b52a04bd5f1bd475f/test/tests/functional/pbs_multi_sched.py#L610